api/v1alpha1/cluster_types.go

```diff
@@ -148,6 +151,9 @@ type ETOSSuiteStarter struct {
 	// +kubebuilder:default={"ttl": "3600", "gracePeriod": "300"}
 	// +optional
 	Config ETOSSuiteStarterConfig `json:"config"`
+	// +kubebuilder:default=1
+	// +optional
+	Replicas *int32 `json:"replicas"`
```

Comment:
To match the API that we are using

```suggestion
	Replicas *int32 `json:"replicas,omitempty" protobuf:"varint,1,opt,name=replicas"`
```